cmd/login_cloud.go

```diff
@@ -14,37 +16,96 @@ var loginCloudCommand = &cobra.Command{
 	Short: "Authenticate with Load Impact",
 	Long: `Authenticate with Load Impact.
 
-This will set the default server used when just "-o cloud" is passed.`,
+This will set the default token used when just "k6 run -o cloud" is passed.`,
```

Comment:
I think we should be more descriptive here, maybe:
"By authenticating you tell k6 to store your account API token in a configuration file on disk for subsequent use when running "k6 run -o cloud"."

Reply:
...and this should be updated to mention cloud execution later.